libraries/AP_ADC/AP_ADC_ADS1115.h

```diff
@@ -6,6 +6,8 @@
 
 #include "AP_ADC.h"
 
+#define ADS1115_I2C_BUS             1
```

Comment:
This ADS1115_I2C_BUS isn't used anywhere.

Can you change this to something like the following?
```
#ifndef ADS1115_I2C_BUS
  #define ADS1115_I2C_BUS 0
#endif
```
I would then add the ADS1115_I2C_BUS definition to the defines in AP_Hal/board/linux.h.